package.json

```diff
@@ -30,7 +39,7 @@
     "prepublishOnly": "npm run clean && npm run lint && npm run test && npm run build",
     "format": "prettier --write {src,test,typescript_test}/**/*.{js,ts}",
     "format:check": "prettier --check {src,test,typescript_test}/**/*.{js,ts}",
-    "lint": "eslint '{src,test,typescript_test}/**/*.{js,ts}'",
+    "lint": "eslint {src,test,typescript_test}/**/*.{js,ts}",
```

Comment:
This reversed my change from https://github.com/reduxjs/redux-thunk/pull/339. Any reason for that?

Reply:
@mgol : hmm. It doesn't seem to work on Git Bash on Windows, which is what _I_ normally use :)

Wonder if escaping double quotes would work better here?

Reply:
Ah, the choose your poison dilemma… 😅 

Yes, double quotes work on macOS as well! I’ve just checked.